samples/Quickstarts/2_InteractiveAspNetCore/src/IdentityServer/Program.cs

```diff
@@ -32,7 +32,7 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args)
                             .MinimumLevel.Override("System", LogEventLevel.Warning)
                             .MinimumLevel.Override("Microsoft.AspNetCore.Authentication", LogEventLevel.Information)
                             .Enrich.FromLogContext()
-                            .WriteTo.File(@"identityserver4_log.txt")
+                            //.WriteTo.File(@"identityserver4_log.txt")
```

Comment:
Zombie code as Serilog.Sinks.File isn't included in packagereference. Any reason to keep it?